journal_name.py

```diff
@@ -3,33 +3,50 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import json
+import pdb
+import types
 
+def store(data,file_name):
+    with open(file_name, 'w') as json_file:
+        json_file.write(json.dumps(data))
+
+        
 def get_json():
```

Comment:
This is not a good approach to implement this functionality. In this same program, you save the JSON file, and then read it again, and then save another JSON.

Please change your implementation approach to have `get_json` function take input parameter with the input as string, and return json dictionary. Don't save json string and then read and save dictionary json. Pass the bibtex string you get from Google Scholar to the function, and save just the result dictionary.